test/src/index.js

```diff
@@ -13,9 +12,5 @@ import './dns.js'
 import './backend.js'
 import './crypto.js'
 import './util.js'
-
-// TODO(@jwerle): FIXME
-if (os.platform() !== 'win32') {
-  await import('./runtime.js')
-  await import('./fs.js')
-}
+import './runtime.js'
```

Comment:
we had this check here for windows, why is it removed? imports are not synchronous so with

Reply:
I moved these checks to the modules. Tests may end with `exit 0` early without this like [here](https://github.com/socketsupply/socket/actions/runs/4099615181/jobs/7069821754). There are failing tests, but no exit 1 as runner exits on some of previous ticks

Reply:
so this gives a false positive